src/thor/timedistancematrix.cc

```diff
@@ -230,6 +230,9 @@ void TimeDistanceMatrix::ComputeMatrix(Api& request,
     SetOrigin<expansion_direction>(graphreader, origin, time_info);
     SetDestinationEdges();
 
+    // Collect edge_ids used for settling a location
```

Comment:
I'd mention that we only need this to determine the right timezone for each destination